src/flow/ui/FlowStackNavigator.tsx

```diff
@@ -49,23 +59,34 @@ const FlowStackNavigator = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, []);
 
-  // When the initial route name is still null, don't return anything.
-  if (!initialRouteName) {
-    return null;
-  }
-
   // Every time the initial route name changes to a different string, reconstruct the entire thing.
   // Why? The initial route name will only be set once(!), i.e. during the initial render.
   // Subsequent changes won't alter the initial route.
   return (
-    <FlowStack.Navigator initialRouteName={initialRouteName}>
-      {FLOW_STACK_SCREEN_ROUTES.map((route) => {
+    <FlowStack.Navigator>
+      {getFlowStackScreenRoutes(initialRouteName).map((route) => {
         const screen = FLOW_STACK_SCREENS[route];
         return (
```

Comment:
# Authentication Flow technique

Retain the stack navigator. Don't destroy it when `initialRouteName` is `null`.

## References

https://reactnavigation.org/docs/6.x/auth-flow

[1/2]